src/pages/experimental/uploading/index.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: Uploading
+---
+
+### Transferring a resource from a local system to a remote system.
+
+<ImageComponent cols="8">
+
+![Contextual image](images/1.png)
```

Comment:
I commented about this on your other PR, but we want to use more detailed alt text for visually impaired readers using screen readers.